### ISSUE TITLE
Do XNOR models take longer to train?

### ISSUE THREAD

**Issue by gnefihs** [AlexeyAB/darknet#3913]
_First of all, I just want to say this repo has been extremely useful for me. I'm rather new to this field and I've learned a lot from this forum. Thank you all and I hope to eventually contribute to this!_

I'm trying to train YOLOv3-tiny-XNOR on a subset of MS COCO (only images with 6 out of the 80 classes). The loss function remains very high even after 20,000 iterations.

My GPU resource is limited so I want to check whether this is normal, before I continue training.

![chart](https://user-images.githubusercontent.com/46212010/64776645-32be1800-d58b-11e9-8664-a5dedece120c.png)

YOLOv3-tiny trains fine, so I don't think my dataset is the problem.
![chart](https://user-images.githubusercontent.com/46212010/64776718-52554080-d58b-11e9-9846-77ab6acf11d0.png)

**Comment by AlexeyAB** [comment]
XNOR model isn't released: https://github.com/AlexeyAB/darknet/issues/3054

Try to train this model:
https://github.com/AlexeyAB/darknet/files/2853459/yolov3-spp_xnor_obj.cfg.txt

By using this pre-trained weights file:
https://drive.google.com/open?id=1IT-vvyxRLlxY5g9rJp_G2U3TXYphjBv8
